libpod/kube.go

```diff
@@ -468,11 +468,28 @@ func generateKubeSecurityContext(c *Container) (*v1.SecurityContext, error) {
 		return nil, err
 	}
 
+	var selinuxOpts v1.SELinuxOptions
+	opts := strings.SplitN(c.config.Spec.Annotations[InspectAnnotationLabel], ":", 2)
+	if len(opts) == 2 {
+		switch opts[0] {
+
+		case "user":
+			selinuxOpts.User = opts[1]
+		case "role":
+			selinuxOpts.Role = opts[1]
+		case "type":
+			selinuxOpts.Type = opts[1]
+		case "level":
+			selinuxOpts.Level = opts[1]
+		default:
+			return nil, errors.Errorf("unknown field of SELinuxOptions: %s", opts[0])
```

Comment:
Should be a `errors.Wrapf(define.ErrInternal, ...`

Reply:
We are also missing `filesystem` support, I think.

Reply:
Oh, sorry, that second comment should go with the one above

Reply:
> We are also missing `filesystem` support, I think.


the fstype is not defined under v1.SELinuxOptions, should it be supported somewhere else?

Reply:
I'm not sure, but we have to support it somehow - otherwise we'll break `generate kube` on containers with it set